### PR TITLE
Added testing for presence of valid .ogg files

### PR DESCRIPTION
**Pull request by atulsaswat**
In reference to a valid ogg file referenced from the samples.

### CODE REVIEW

File: tests/audio/test_loading.py [hunk 1]
```diff
@@ -3,3 +3,7 @@
 def test_missing_file(audiomgr):
     sound = audiomgr.get_sound('/not/a/valid/file.ogg')
     assert str(sound).startswith('NullAudioSound')
+
+def test_found_file(audiomgr):
```
Comment (Moguri) [inline tests/audio/test_loading.py]: Let's rename this to `test_load_ogg_vorbis()` to make adding other file types (e.g., ogg opus) more obvious.

Comment (Moguri) [reply tests/audio/test_loading.py]: This path is relative to the current working directory; we should make it relative to `__file__` using something like:

```python
import panda3d.core
path = p3d.Filename.from_os_specific(os.path.join(os.path.dirname(__file__), '..', '..', 'samples', 'music-box', 'music', 'musicbox.ogg'))
```

Or, maybe a little cleaner:
```python
path = os.path.dirname(__file__) + '../../samples/music-box/music/musicbox.ogg'
```

Comment (atulsaswat) [reply tests/audio/test_loading.py]: Okay, Got it.

**Comment by Moguri** [comment]
@rdb Will the test still fail (as written) if the loader failed to load the audio file? For example, if the codec was not supported?

**Comment by rdb** [comment]
Hi Atul, are you planning to come back to this or should this be closed?

**Comment by Moguri** [comment]
Closing this due to inactivity.

@atulsaswat feel free to re-open this if you're still intending to work on this.

**Pull request closed without merging**